shiva/src/main/scala/nl/lumc/sasc/biopet/pipelines/shiva/variantcallers/HaplotypeCallerGvcf.scala

```diff
@@ -124,6 +124,7 @@ class HaplotypeCallerGvcf(val parent: Configurable) extends Variantcaller {
     }
 
     genotypeGvcfs.outputDir = outputDir
+    genotypeGvcfs.namePrefix = s"$namePrefix.$name"
```

Comment:
Not directly. This name prefix is for genotypeGvcfs, the other is from HaplotypecallerGvcf. Only in HaolotpecallerGvcf I do set the namePrefix of GenotypeGvcfs